specification/mongocluster/DocumentDB.MongoCluster.Management/MongoCluster.tsp

```diff
@@ -362,3 +409,56 @@ union ReplicationState {
   /** Replication link is re-configuring due to a promotion event. */
   "Reconfiguring",
 }
+
+/** Specification for a node group. */
+@removed(Versions.v2024_07_01)
+model NodeGroupSpec is NodeGroupProperties {
+  /** The node type deployed in the node group. */
+  kind?: NodeKind;
+
+  /** The number of nodes in the node group. */
+  nodeCount?: int32;
+}
+
+/** The properties of the node group on a cluster. */
+@removed(Versions.v2024_07_01)
+model NodeGroupProperties {
+  /** The resource sku for the node group. This defines the size of CPU and memory that is provisioned for each node. Example values: 'M30', 'M40'. */
+  sku?: string;
+
```

Comment:
sku should be a top level property

Reply:
This is an existing property definition on previous versions of the API - is the ask to move the typespec properties but leave the API specification unchanged?

Additionally, 'sku' here is not Azure Compute sku definitions but a sku definition that are used to describe mongo clusters i.e it is not 'Standard_D4ds_v5' but uses 'M30', 'M40' sku names.

Reply:
@olivertowers This model has the `@removed` annotation so it's no longer applicable to this API version, correct?

Reply:
That's correct, for the first stable API we are refactoring the model away from the structure we had in our preview releases.

The new stable API is essentially a refactor only of the MongoCluster properties with no real net new 'features' being added.

Reply:
given the explanation u may ignore my comment